tests/unit/test_ctr.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from contxt.models.changetrackingrecord import ChangeTrackingRecord, CTRHeader
+
+
+def test_ctr_wrong_topic():
+    with pytest.raises(AttributeError):
+        ChangeTrackingRecord(404, 1020304, 0, "soruce", "table", "db", {"a": 1, "b": 2, "c": 3})
```

Comment:
note, you're not passing in the `DBOperation` enum you defined when instantiating these objects. that would be DBOperation.INSERT rather than 0